src/qmltypes/qmlfilter.cpp

```diff
@@ -691,7 +689,7 @@ void QmlFilter::preset(const QString &name)
         m_service.load(qUtf8Printable(fileName));
     }
 
-    emit changed();
+    emit changed("");
```

Comment:
Another change to remove after changing the signal signature.

Reply:
done